core/src/epicli/data/common/defaults/configuration/applications.yml

```diff
@@ -119,13 +119,16 @@ specification:
         PGPOOL_ADMIN_USERNAME: epi_pgpool_admin # Pgpool administrator (local pcp user)
         PGPOOL_ENABLE_LOAD_BALANCING: true # set to 'false' if there is no replication
         PGPOOL_MAX_POOL: 4
+        PGPOOL_CHILD_LIFE_TIME: 0
         PGPOOL_POSTGRES_PASSWORD_FILE: /opt/bitnami/pgpool/secrets/pgpool_postgres_password
         PGPOOL_SR_CHECK_PASSWORD_FILE: /opt/bitnami/pgpool/secrets/pgpool_sr_check_password
         PGPOOL_ADMIN_PASSWORD_FILE: /opt/bitnami/pgpool/secrets/pgpool_admin_password
+        PGPOOLKEYFILE: /opt/bitnami/pgpool/secrets/pgpool_sha_encryption_key
       secrets:
         pgpool_postgres_password: PASSWORD_TO_CHANGE
         pgpool_sr_check_password: PASSWORD_TO_CHANGE
         pgpool_admin_password: PASSWORD_TO_CHANGE
+        pgpool_sha_encryption_key: PASSWORD_TO_CHANGE  # used only if SCRAM SHA-256 passwords encryption enabled
```

Comment:
If we are going to leave md5 as default I would comment it out and put into documentation information about the possibility of using SCRAM